slice/Ice/Ice2Definitions.ice

```diff
@@ -28,6 +28,12 @@ module Ice
     {
     }
 
+    /// The keys for supported Ice2 connection parameters.
+    unchecked enum Ice2ParameterKey : int
+    {
+        OutgoingFrameMaxSize = 0
```

Comment:
I think it would be more logical to call this parameter `IncomingFrameMaxSize`: the sender provides its IncomingFrameMaxSize, and the recipient interprets this size as its own OutgoingFrameMaxSize.